lute/io/models/sfx_index.py

```diff
@@ -403,3 +403,19 @@ def validate_out_file(cls, out_file: str, values: Dict[str, Any]) -> str:
             fname: str = f"{expmt}_r{run:04d}.stream"
             return f"{work_dir}/{fname}"
         return out_file
+
+
+class MergeStreamFilesParameters(TaskParameters):
+
+    in_file: Optional[str] = Field(
```

Comment:
Maybe this should be named `file_dir` or similar? It's the directory, not a file right? Otherwise the `Task` should probably handle the case where a single file is provided.

Reply:
It's actually, as it is now, after the last changes, a file. With a file named `path/tag_name.stream`, this task will look for all files in `path/tag_*.stream` and output a `path.stream` file

Reply:
True, lets leave it as is then. Thanks!